irohad/consensus/yac/transport/yac_pb_converters.hpp

```diff
@@ -89,9 +89,16 @@ namespace iroha {
 
         static boost::optional<VoteMessage> deserializeVote(
             const proto::Vote &pb_vote, logger::LoggerPtr log) {
+          // TODO IR-428 igor-egorov refactor PbConverters - do the class
+          // instantiable
+          static const uint64_t kMaxBatchSize{0};
+          // This is a workaround for the following ProtoCommonObjectsFactory.
+          // We able to do this, because we don't have batches in consensus.
           static shared_model::proto::ProtoCommonObjectsFactory<
               shared_model::validation::FieldValidator>
-              factory_;
+              factory_{
```

Comment:
```suggestion
              factory{
```
the `_` suffix is used to mark instance members. But here It hurts my glazzies badly!

Reply:
Did you mean eyees?

Reply:
Viddy well, droogie!